conf/application.conf

```diff
@@ -234,7 +234,7 @@ eligibility-stats {
 # let min-backoff be `m` and max-backoff be `M`. Then `2m-M` cannot equal zero
 uc-threshold {
   # this is a temporary flag to disable the DES theshold behaviour until we are ready in production
-  enabled     = false
+  enabled     = true
```

Comment:
is there a reason why this has changed in this PR?

Reply:
no I don't know how that happened, I've changed it back